pkg/recipes/driver/terraform.go

```diff
@@ -174,9 +177,13 @@ func (d *terraformDriver) prepareRecipeResponse(ctx context.Context, definition
 		TemplateVersion: definition.TemplateVersion,
 	}
 
-	deployedResources, err := d.getDeployedOutputResources(ctx, tfState.Values.RootModule)
-	if err != nil {
-		return &recipes.RecipeOutput{}, err
+	var deployedResources []string
+	if tfState.Values != nil && tfState.Values.RootModule != nil {
```

Comment:
nit: we are already doing the nil check for `RootModule` inside d.getDeployedOutputResources()
```suggestion
	if tfState.Values != nil {
```

Reply:
It's GoLang, you can never have too many nil checks